.storybook/themes/commerce.scss

```diff
@@ -104,58 +104,58 @@
 		"font-weight-8": 800,
 		"font-weight-9": 900,
 
-		"font-size-1": 0.625rem,
 		// 10px
-		"font-size-2": 0.75rem,
```

Comment:
can you revert some of these font-size changes? If this is keeping it in line with something else, we can maybe make it a different pr?